src/app/core/common-components/entity-select/entity-select.component.ts

```diff
@@ -135,6 +135,14 @@ export class EntitySelectComponent<
 
   @Input() additionalFilter: (e: E) => boolean = (_) => true;
 
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes["form"]) {
+      this.form.valueChanges.subscribe((value) => {
+        this.updateAvailableOptions().then((_) => {});
+      });
+    }
+  }
+
```

Comment:
@tomwwinter @Abhinegi2  is this a change one of you did somewhere for a bugfix? I can't remember why this should be here anymore :see_no_evil:

Reply:
As far as I remember 😅, I did not make these changes.